pineapple-pictures.json

```diff
@@ -0,0 +1,70 @@
+{
+    "app-id": "net.blumia.pineapple-pictures",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15-22.08",
+    "sdk": "org.kde.Sdk",
+    "command": "ppic",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
```

Comment:
if it supports wayland then line 9 should be fallback-x11 instead.

Reply:
patch updated :)